docs/how-to/manage-identities.md

```diff
@@ -0,0 +1,127 @@
+
+# How to manage identities
+
+Pebble uses named [identities](../reference/identities.md) to extend access to its [API](../explanation/api-and-clients.md).
+
+By default any UID (user ID) connected to the API socket is a `read` user; UID 0 (root) or the UID of the Pebble server process gives `admin` access.
+
+To extend this to additional users, add named identities using the [`add-identities`](../reference/cli-commands/add-identities.md) command with a YAML file configuring the details. For example, to add a new admin "bob" with UID 42 and a new read user "alice" with UID 2000, prepare this file:
```

Comment:
I think it's better to paint a picture here, of what problem we are trying to solve here. Why the users might want to extend this to additional users, why not run the Pebble daemon and API with the same user, why not run API with the root user, etc?

A picture in my mind could be:

Imagine you have a cross-functioning team working on the same product on the same range of servers, Ops might need to deploy stuff so they need admin access, and developers might need read access to monitor and debug services. You could allow everybody to SSH using the same user with which the daemon is running, but sharing accounts is not a good practice. You could also let everybody run API with root to bypass the permission issue, but this violates the principle of least privilege in security. To solve this problem, you can use Pebble's identity and access management feature.

Reply:
Thanks @IronCore864 ! I agree it would be helpful to first provide a bit of context as to when you would want to use identities in Pebble, and how that would help your real-world scenario.

As to the current format, this is something to which I have a slightly different opinion compared to existing doc sets.

I think all the different options for `pebble *identities*` don't actually need to be in how-tos but can be kept in the Reference doc for Identities, with links to the relevant CLI commands.
Why? Because all those options describe the machinery / tool, what Pebble can do.
See this part "Refer to the x reference guide for a full list of options.".
https://diataxis.fr/how-to-guides/#the-language-of-how-to-guides

If we go back to the very basic analogy, it's like a recipe.
https://diataxis.fr/how-to-guides/#applied-to-food-and-cooking
So what is the user trying to accomplish?

But I know it's a difficult question to address :) 

As a start I would suggest the following changes (the rest can be improved later on as needed)
1. Provide a bit more context as @IronCore864 suggested so that when a user lands on this page, they have an idea how / when they would want to manage / use Identities in Pebble.
2. Structure each section with relevant headings so it's easier to navigate. It might seem repetitive but it makes it a lot easier to get to where you want. E.g. "How to manage identities", "Add a new identity", "Remove an identity" etc. 
3. I would suggest moving the "list identities" command before you update / remove / delete since you would (presumably) need to verify what exists before you try to change it.

Hope this makes sense, thanks!

Reply:
Please ignore my suggestion about (potentially) moving this to Reference. :)

Reply:
Thanks. I've added a paragraph with a scenario. It's not quite as "big" as Tiexin's suggestion, but hopefully it servers as a context-setting introduction.

I've also added headings.

@AnneCYH Regarding moving "list identities" to the top, the problem with that is that at that stage there are no identities to list, so the output isn't useful. So I'd prefer to keep this one till the bottom so that we have some useful stuff to show (especially with the YAML/JSON modes).

Reply:
Thanks @benhoyt , i think the brief intro is sufficient. 
And understood about the List identities sequence. I thought it might be good to List right after you Add before you Update or Remove.
But it's a minor thing.

Thanks!